Towny/src/main/java/com/palmergames/bukkit/config/ConfigNodes.java

```diff
@@ -1277,6 +1277,12 @@ public enum ConfigNodes {
 			"# +------------------------------------------------------+ #",
 			"############################################################",
 			""),
+	SPAWNING_SAFE_TELEPORT(
+		"spawning.safe_teleport",
+		"false",
+		"# If enabled tries to find a safe location when teleporting to a town spawn/nation spawn/outpost",

```

Comment:
This needs one more line after "false", and before the first comment:
```
"",
```
This is how we get the nice spacing in the config.